src/main/java/com/google/sps/util/DataStoreHelper.java

```diff
@@ -0,0 +1,47 @@
+package com.google.sps.util;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.util.Date;
+
+
+import com.google.cloud.datastore.BaseEntity;
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Key;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+
+public class DataStoreHelper {
+
+    public static final Datastore DATASTORE = DatastoreOptions.getDefaultInstance().getService();
```

Comment:
I am not sure if this can be a static. Are you not getting a compile error?

Reply:
I didn't get a compile error when I ran `mvn package exec:java`
Maybe the compile error came from the empty TripEvent class.